mne/viz/montage.py

```diff
@@ -1,16 +1,18 @@
 """Functions to plot EEG sensor montages or digitizer montages."""
-import numpy as np
+import mne
+from .utils import plot_sensors
 
-from .utils import plt_show
 
-
-def plot_montage(montage, scale_factor=20, show_names=False, show=True):
+def plot_montage(montage, kind='3d', scale_factor=20, show_names=False,
```

Comment:
same remark here. Put the kind at the end. Actually please put it just before the show=True as so fare show is always the last param in the plot function signature.